sites/dev/pages/[...slug].mjs

```diff
@@ -1,100 +1,7 @@
-// Used in static paths
+import { DocsPage, ns as pageNs } from 'shared/components/docs/docs-page.mjs'
```

Comment:
Of course I feel the opposite way. This code was basically a duplicate of the one in org, so when I went in to update it, I felt it was easier to maintain consistency if changes only happened in one place. It's true that this is the reason dev won't build now, because dev doesn't have a docs layout, but it's my opinion that that layout should be shared too.

That having been said, I will try to do less abstraction. I may need help understanding where that line is for you